R/utils.R

```diff
@@ -9,6 +9,24 @@ phantom_run <- function(args, wait = TRUE) {
   system2(phantom_bin, args = args, wait = wait)
 }
 
+
+# Try really hard to find bower in Windows
+find_phantom <- function(){
+  # a slightly more robust finder of bower for windows
+  # which does not require PATH environment variable to be set
+  phantom_path = if(Sys.which("phantomjs") == "") {
+    # if it does not find Sys.which('bower')
+    # also check APPDATA to see if found there
+    if(identical(.Platform$OS.type,"windows")) {
+      Sys.which(file.path(Sys.getenv("APPDATA"),"npm","phantomjs."))
```

Comment:
path to `phantom` is actually `phantom.cmd` so appears like this in the directory.  Without the `.` you'll get the nonexecutable version.  Could be more specific with `.cmd` but I don't know enough about `npm` to know side effects.

![image](https://cloud.githubusercontent.com/assets/837910/7140212/ba754884-e28f-11e4-9fd8-30cb429095c0.png)